content/posts/2019-12-12-download-hawk.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Introducing DownloadHawk"
+date: 2019-12-12
+authors:
+- degeri
+tags: [Decred,security,binary verification]
+banner_image: "/images/posts/download_hawk_1.png"
+---
+
+We recently observed the Monero hack in which an attacker compromised getmonero[.]org and replaced one of the binaries with a malicious coinstealer. 
+
+The Monero team has good reliability when it comes to security and all indicators seem to show that the attack was sophisticated. Point being no matter what your security posture, this could happen to anyone.
+
+
+<!--more-->
+
+# How should everyone protect themselves? 
```

Comment:
These should be second level headers (`##` instead of `#`). No other blog post uses first level headers

Reply:
Moved everything down a level.